beetmoverscript/script.py

```diff
@@ -12,16 +12,16 @@
 import aiohttp
 import boto3
 
-from scriptworker.client import get_task, validate_artifact_url
+from scriptworker.client import get_task
 from scriptworker.context import Context
 from scriptworker.exceptions import ScriptWorkerTaskException, ScriptWorkerRetryException
-from scriptworker.utils import (retry_async, download_file,
-                                raise_future_exceptions, retry_request)
+from scriptworker.utils import retry_async, download_file, raise_future_exceptions
```

Comment:
I think we don't call `retry_download` anywhere but in the tests anymore, so we can probably remove it.